app/models/release.rb

```diff
@@ -44,6 +44,9 @@ def self.find_by_version!(version)
   private
 
   def assign_release_number
+    default_number = 1
```

Comment:
should be a constant maybe add a `# detect wether the user has overwritten number via parameters, done via xyz plugin`